src/main/java/app/linguistai/bmvp/controller/ProfileController.java

```diff
@@ -39,4 +44,10 @@ public ResponseEntity<Object> getProfile(@RequestHeader(Header.USER_EMAIL) Strin
         return Response.create("OK", HttpStatus.OK, userProfile);        
     }
 
+    @GetMapping(consumes = MediaType.APPLICATION_JSON_VALUE, path = "search")
+    public ResponseEntity<Object> searchUser(@Valid @RequestBody QUserSearch userSearch,
```

Comment:
@selimcanglr Currently search endpoint's parameters (username, page, size) need to be given in the request body. Would you prefer if they were given as part of the URL?

Reply:
That would be more preferable, it doesn't matter that much.